.github/workflows/build_pr_documentation.yml

```
@@ -80,6 +80,7 @@ jobs:
           path: ${{ inputs.package }}
 
       - name: Checkout to the correct commit
+        if: github.event_name == 'issue_comment'
```

```suggestion
        if: ${{ github.event_name == 'issue_comment' }}
```

(from the example in this part of the docs: https://docs.github.com/en/actions/reference/workflows-and-actions/contexts#example-usage-of-the-github-context)

---

without it , still works, but good to have this format